A/AOCL/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+using BinaryBuilder, Pkg
+
+name = "AOCL"
+version = v"5.1.0"
+sources = [
+    ArchiveSource("https://download.amd.com/developer/eula/blas/blas-5-1/aocl-blis-linux-gcc-5.1.0.tar.gz",
```

Comment:
Instead of hardcoding the versions, better to do this?
```suggestion
    ArchiveSource("https://download.amd.com/developer/eula/blas/blas-$(version.major)-$(version.minor)/aocl-blis-linux-gcc-$(version).tar.gz",
```
and similar for the other URLs